cloudmapper.py

```diff
@@ -49,10 +49,27 @@ def get_account(account_name, config, config_filename):
 def run_gathering(arguments):
     from cloudmapper.gatherer import gather
     parser = argparse.ArgumentParser()
+    parser.add_argument("--config", help="Config file name",
+                        default="config.json", type=str)
     parser.add_argument("--account-name", help="Account to collect from",
-                        required=True, type=str)
+                        required=False, type=str)
+    parser.add_argument("--profile-name", help="AWS profile name",
+                        required=False, type=str)
+    parser.add_argument('--clean', help='Remove any existing data for the account before gathering', action='store_true')
```

Comment:
I'm wondering if now, with functionality like this, I should add some checks to ensure a user doesn't name their account `/home` and then end up running `--clean` on that account and end up deleting a bunch of data.  I'll add in checks later on the account names.

Reply:
So, I kept the "./" in the paths, just moved it to a variable. This should not be an issue.  https://github.com/alanquillin/cloudmapper/blob/904c08861735062e91518aba644f64b8d83c291d/cloudmapper/gatherer.py#L15